docs/connectors/pre-built-connectors.md

```
@@ -13,9 +13,9 @@ title: "Pre-built Connectors"
 | Cerner | EHR | Database | [Link](https://github.com/tuva-health/cerner_connector) |
 | CMS BCDA | Claims | FHIR | [Link](https://github.com/tuva-health/bcda_connector) |
 | CMS CCLF | Claims | Flat File | [Link](https://github.com/tuva-health/medicare_cclf_connector) |
-| CMS LDS | Claims | Flat File | [Link](http://localhost:3000/connectors/cms-lds) |
+| CMS LDS | Claims | Flat File | [Link](https://github.com/tuva-health/medicare_lds_connector) |
```

markdownlint-cli2 (0.18.1)

Link text should be descriptive

(MD059, descriptive-link-text)